src/stacktrace.rs

```diff
@@ -254,5 +265,13 @@ impl Filter for Stacktrace {
                 && (entry.module.is_empty() || !rfile.is_match(&entry.module))
                 && (entry.debug.file.is_empty() || !rfile.is_match(&entry.debug.file))
         });
+
+        // For libfuzzer: delete functions below LLVMFuzzerTestOneInput
+        if let Some(pos) = &self
```

Comment:
This should be performed prior to regex filtering because regex filters also contain LLVMFuzzer...

Reply:
done